Src/Support/CommonPackageInfo.xml

```diff
@@ -0,0 +1,12 @@
+<Project>
+  <PropertyGroup>
+    <PackageVersion>1.25.0</PackageVersion>
+	<Authors>Google Inc.</Authors>
+	<Copyright>Copyright 2017 Google Inc.</Copyright>
```

Comment:
We haven't been including this in other packages (GAX etc). That might have been a mistake.